back/spec/services/json_forms_service_spec.rb

```diff
@@ -240,16 +250,21 @@
           key: 'idea_files_attributes',
           code: 'idea_files_attributes'
         )
-        fields = [required_field, optional_field, build_in_required_field, build_in_optional_field]
+        fields = [built_in_main_field, required_field, optional_field, build_in_required_field, build_in_optional_field]
 
-        output = service.input_ui_and_json_multiloc_schemas fields, user
+        output = service.input_ui_and_json_multiloc_schemas fields, user, 'question'
         expect(output).to include(
           {
             json_schema_multiloc: {
               'en' => {
                 type: 'object',
                 additionalProperties: false,
                 properties: {
+                  'title_multiloc' => {
+                    type: 'object',
+                    minProperties: 1,
+                    properties: { 'en' => { type: 'string', minLength: 10, maxLength: 80 } }
```

Comment:
I think we'll have to solve an issue when we would use the schema to validate inputs on the backend side. It is possible to machine translate a whole platform. When the translation does not meet the character limit, this will give issues with tenant creation.

Reply:
Correct. Max lengths are problematic when doing machine translations.